libraries/client/client_webhook.py

```diff
@@ -187,6 +192,12 @@ def process_webhook(self):
             source_url = self.build_multipart_source(file_key, book)
             identifier, job = self.send_job_request_to_tx_manager(commit_id, source_url, rc, repo_name, repo_owner,
                                                                   count=book_count, part=i, book=book)
+
+            # Send lint request to tx-manager
+            lint_results = self.send_lint_request_to_tx_manager(job, source_url)
```

Comment:
I understand. Shouldn't need a master job. I didn't really look at how you handled errors and warnings until now...I did want you did with concatenating the errors, but did so also for 'warnings'. Is there a reason the 'info' (or is it it just 'log') array was also not concatenated?